CHANGELOG.md

```diff
@@ -50,6 +50,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - Removed `get_node_settings` function as it is no longer used.
   - Renamed `get_customer_id_of_review_host` to `get_customer_id_of_node`.
 - Updated review-database to 0.29.1.
+- Use `StringNumber` for integers beyond `i32`
+  - In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, and `TidbRule`, 
+    changed GraphQL APIs to return `StringNumber` or `ID` instead of integers beyond `i32`.
```

Comment:
Because the first line doesn't mention `ID` and the second includes the first, the first doesn't look necessary to me.